### PR TITLE
Disables browser caching, All deletion now class

### PR DESCRIPTION
**Pull request by Apogate**
- Disables client-side caching using meta tags, should fix some issues.
  Delete_post and usrdel() are now in the Delete class.

### CODE REVIEW

File: imgboard.php [hunk 1]
```diff
@@ -210,39 +210,14 @@ function delete_post( $resno, $pwd, $imgonly = 0, $automatic = 0, $children = 1,
...
-    error( S_BADDELPASS );*/
-    log_cache();
-    //mysql_board_call("UNLOCK TABLES");
-    foreach ( $rebuild as $key => $val ) {
-        updatelog( $key, 1 ); // leaving the second parameter as 0 rebuilds the index each time!
-    }
-    if ( $rebuildindex )
-        updatelog(); // update the index page last
+	require_once(CORE_DIR . "/log/log.php");
+	require_once(CORE_DIR . "admin/delpost.php");
```
Comment (RePod) [inline imgboard.php]: "/admin

Comment (Apogate) [inline imgboard.php]: you're not my real dad

**Comment by Apogate** [comment]
welp, thats all i got for tonight

File: _core/general/head.php [hunk 2]
```diff
@@ -36,7 +36,8 @@ function generate() {
                 <meta name='description' content='" . S_DESCR . "'/></meta>
                 <meta http-equiv='content-type'  content='text/html;charset=utf-8' /></meta>
                 <meta name='viewport' content='width=device-width, initial-scale=1'></meta>
-                <meta http-equiv='pragma' content='no-cache'></meta>
+                <META HTTP-EQUIV=\"Pragma\" CONTENT=\"no-cache\">
+				<META HTTP-EQUIV=\"Expires\" CONTENT=\"-1\">
```
Comment (RePod) [inline _core/general/head.php:40]: Why are these escaped?

**Pull request merged**